test/test_repolib.py

```diff
@@ -880,3 +884,114 @@ def test_empty_manage_repos_int_37(self):
         manage_repos_enabled = repolib.manage_repos_enabled()
         # Should fail, and return default of 1
         self.assertEqual(manage_repos_enabled, True)
+
+
+AUTO_ENABLE_YUM_PLUGINS_ENABLED = """
+[rhsm]
+auto_enable_yum_plugins = 1
+"""
+
+AUTO_ENABLE_YUM_PLUGINS_DISABLED = """
+[rhsm]
+auto_enable_yum_plugins = 0
+"""
+
+YUM_PLUGIN_CONF_FILE_ENABLED = """
+[main]
+enabled = 1
+"""
+
+YUM_PLUGIN_CONF_FILE_DISABLED = """
```

Comment:
Perhaps add a test case for a plugin configuration file that does not match expected values.
`YUM_PLUGIN_CONF_FILE_INVALID = """
[main]
enabled = 4
"""

Reply:
ACK.